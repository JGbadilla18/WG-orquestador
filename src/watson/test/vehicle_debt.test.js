const testWatson = require('./testWatson');

watsonText = 'Prestamos de vehiculos'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
