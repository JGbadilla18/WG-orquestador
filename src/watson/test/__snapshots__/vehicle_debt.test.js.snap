// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prestamos de vehiculos 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Aquí",
          "url": " https://blog.corporacionbi.com/crediauto-en-linea-banco-industrial",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Los requisitos para optar a un crédito de vehículo son: 
  *  Constancia de ingresos
  *  Fotocopia DPI
  *  Copia del carné del NIT
  *  Últimos 3 estados de cuenta donde refleje tus ingresos
  *  Recibo de servicios (agua, luz o teléfono - no celular -) de los últimos 3 meses

 Cotiza y solicita tu préstamo ",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
